### DISCUSSION TITLE
Value calculation on Fungibles - Token extension considerations

### DISCUSSION THREAD

**Discussion by jsblair9** [phantom/docs#193] [General]
I'm thinking through a few scenarios where it may be useful to use token extensions in order to properly display the value of the token in the wallet (and to be accurately reflected in the total calculation). 

I understand the order of operation is the Coingecko price, and then now Birdeye - but if those come back null... is it possible for fallback data to be considered from a metadata pointer? I know even as I type it out what a can of worms I'm suggesting we open here... but would love to discuss if it is considerable

**Comment by adamdelphantom** [comment]
Hey @jsblair9,

The best path forward would be to post your feature request [here.](https://phantom.canny.io) The more upvotes the request gets the better chance it has of getting implemented.

**Reply by jsblair9** [reply]
thanks Adam - after more thought, this is a horrendously bad idea that opens up so much more bad than good 😂
